CEPCPreamble.md

```diff
@@ -0,0 +1,40 @@
+This might mean allowing a new person to become the AC rep. This might mean paying attention to how much time you spend talking in a meeting or how often you join the queue. It might mean sharing slides a few days before a meeting so that people have an opportunity to read and translate them into their native languages. Inclusion is about making people feel welcome. 
+
+3. Think of others’ needs from their point of view. Use preferred names, titles (including pronouns), and the appropriate tone of voice. Therefore, be formal and conservative in what you do and liberal in what you accept from others and acknowledge the contributions of your peers. 
+It is impossible to be aware of all of someone else's needs without asking them what they need. This can be as simple as how to pronounce a name or more subtle, like recognizing that some people prefer to contribute by listening. Avoid making assumptions others' preferences and needs. When in doubt, ask, and respect the response. It's important to recognize the contributions of others. 
+
+Unaccetpable behaviors include:
+4. Offensive comments related to socio-economic status, sexual orientation, religion, race, physical appearance, neurotype, nationality, mental health, language, indigeneity, immigration status, gender, gender identity and gender expression, ethnicity, disability (both visible and invisible), caste, body, or age.
+It is never acceptable to say something offensive. You might not realize that your comments are offensive. This code priorotizes impact of statements over intent. You might not have intended a comment to be offensive, but it can still be hurtful. In such a situation, an apology or correction is still be called for.
+
+5. Misgendering someone by deliberately referring to a person using the wrong pronouns or by using someone's proper names or other terms that person has asked not to be used, also known as deadnaming.
+When someone provides a name to refer to them by, respect the information they provide about themselves even if it is different from information with which you are familiar. Someone may have gone through a recent life change that brought about a name change. Someone may have recently married, divorced, or transitioned. 
+
+6. Stalking or physically following or invading someone's personal space after a request to stop.
+
+Respect people's personal space, especially if they've expressed a preference for a boundary. Different cultures and individuals have different tolerances for what is considered an appropriate amount of space when talking. Some people prefer to avoid any touch. So much more so, do not follow others. Stalking is an extreme form of this that is certainly not acceptable.

```

Comment:
This sentence seems garbled: "So much more so, do not follow others."  IDK what it should be.